po/es.po

```diff
@@ -3,15 +3,16 @@ msgstr ""
 "Project-Id-Version: newsboat 2.5\n"
 "Report-Msgid-Bugs-To: https://github.com/newsboat/newsboat/issues\n"
 "POT-Creation-Date: 2021-12-08 18:49+0300\n"
-"PO-Revision-Date: 2020-04-21 23:43+0200\n"
-"Last-Translator: Marcos Cruz <github@programandala.net>\n"
+"PO-Revision-Date: 2022-01-31 18:32+0100\n"
+"Last-Translator: Roberto Michán Sánchez <robertoms258@gmail.com>\n"
 "Language-Team: ROOT <epsilon@correoe.no-ip.org>, OmeGa <omega@mailoo.org>, "
 "José Manuel García-Patos <josemanuel@gesaku.es>\n"
```

Comment:
Marcos should probably be added to the `Language-Team`, so that he's still credited for his work. You might also add yourself to that list, at least in most other translations the `Last-Translator` is also part of the `Language-Team`. But I'm actually not entirely sure on our convention here.

Reply:
Alright, I added both of us to the list in the following commit, and also addressed the mistaken quote previously mentioned.